templates/default/nrpe.service.erb

```diff
@@ -1,17 +1,21 @@
 [Unit]
-Description=NRPE
-After=network.target
-Requires=network.target
+Description=Nagios Remote Program Executor
+Documentation=http://www.nagios.org/documentation
+Conflicts=nrpe.socket
+Requires=network-online.target
+After=var-run.mount nss-lookup.target network.target local-fs.target time-sync.target
+Before=getty@tty1.service xdm.service
+
+[Install]
+WantedBy=multi-user.target
 
 [Service]
-Type=forking
+Type=simple
 User=<%= @nrpe['user'] %>
 Group=<%= @nrpe['group'] %>
 <% if ['rhel', 'fedora'].include?(node['platform_family']) && ::File.exist?('/etc/sysconfig/nrpe') -%>
 EnvironmentFile=/etc/sysconfig/nrpe
 <% end -%>
-ExecStart=/usr/sbin/nrpe -c <%= @nrpe['conf_dir'] %>/nrpe.cfg -d $NRPE_SSL_OPT
```

Comment:
Are we not allowing this to be configurable anymore?

Reply:
Good catch!  I'll fix that